### PR TITLE
[RELEASE NOTES] Known issue azure-eventhub in Windows / 8.15

### PR DESCRIPTION
**Pull request by lucabelluccini**
Update the Release notes to include known issue

- https://github.com/elastic/beats/issues/40608
- https://github.com/elastic/beats/pull/40609

### CODE REVIEW

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -7,6 +7,12 @@
 === Beats version 8.15.0
 https://github.com/elastic/beats/compare/v8.14.3\...v8.15.0[View commits]
 
+==== Known issues
+
+*Filebeat*
+
+- The Azure EventHub input in Filebeat is not found when running on Windows. Please refrain upgrading to 8.15. See {issue}40608[40608]
```
Comment (kilfoyle) [inline CHANGELOG.asciidoc:14]:
```suggestion
- The Azure EventHub input in Filebeat is not found when running on Windows. Please refrain from upgrading to 8.15. See {issue}40608[40608] for details.
```

**Review by kilfoyle — APPROVED** [review]
LGTM! 🚀 
Just a small suggested change for the wording.

@pierrehilbert  Please have a look too as I don't really know much about the Beats Release Notes.

**Comment by lucabelluccini** [comment]
Thanks @kilfoyle - Idk how to format this to show up as the fleet PR you've done https://github.com/elastic/ingest-docs/pull/1268

**Review by pierrehilbert — APPROVED** [review]

**Comment by kilfoyle** [comment]
> Idk how to format this to show up as the fleet PR you've done

@lucabelluccini  The Beats release notes are a really different format from the Fleet & Agent ones, so I think what you've added here is perfect.

**Comment by kilfoyle** [comment]
I think this can be merged now...

**Pull request merged**